bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js

```diff
@@ -0,0 +1,113 @@
+import { StateHandler, controllerMixin } from 'oskari-ui/util';
+import { createFeaturedataGrid, showFeatureDataFlyout } from './FeatureDataFlyout';
+import { getSorterFor } from '../../../../src/react/components/Table';
```

Comment:
En muista saako getSorteria suoraan oskari-ui:n kautta, mutta olisi hyvä hakea se jotenkin sen kautta kuitenkin.
```suggestion
import { getSorterFor } from 'oskari-ui/Table';
```